### PR TITLE
Added env-cmd package to handle multiple env files

### PR DESCRIPTION
**Pull request by asimpk**

### CODE REVIEW

**Review by udosson — COMMENTED** [review]
Did you also change the `.env` file at the root level of the repository?

**Comment by asimpk** [comment]
> Did you also change the `.env` file at the root level of the repository?

No, only changed in the `dev-frontend` package.

**Comment by udosson** [comment]
Is there a `.env` in your local branch with the ENVs to connect to Polygon?

File: packages/dev-frontend/package.json [hunk 1]
```diff
@@ -48,7 +49,8 @@
   },
   "scripts": {
     "analyze": "source-map-explorer 'build/static/js/*.js'",
-    "start": "react-scripts start",
+    "start": "env-cmd -f .env react-scripts start",
```
[the remark this answers is not in the document]
Comment (asimpk) [reply packages/dev-frontend/package.json:52]: > `.env` where is this file?
Oh Thanks actually `.env` is in `dev-frontend` package and yeah it's added in .gitignore that's why not showing here.
Connecting with Polygon endpoints settings are also in default `.env` 
What you say, how we merge those changes? I need to remove it from gitignore and push that changes too.

[the remark this answers is not in the document]
Comment (udosson) [reply packages/dev-frontend/package.json:52]: You can use `git add --force pathToEnvFile/.env`
Please check the `.env` for sensitive information before pushing it. 
Thanks

**Review by udosson — APPROVED** [review]
LGTM. Thanks @asimpk 
Could you please create a readme file in `packages/front-end` that explains how to use it? It's basically the information you already provided via discord

**Pull request merged**